flink-core/src/test/java/org/apache/flink/util/FileUtilsTest.java

```diff
@@ -202,6 +202,24 @@ public void testDeleteDirectoryWhichIsAFile() throws Exception {
 		}
 	}
 
+	@Test
+	public void testDeleteSymbolicLinkDirectory() throws Exception {
+
+		// deleting a symbolic link directory should not delete the files in it.
+
+		File symbolicLinkDirectory = tmp.newFolder();
```

Comment:
When I first read the test, I didn't understand it. Maybe add a comment `// creating a directory to which the test creates a symbolic link` or would `linkedDirectory` as a variable name clearer?